doc/api/cli.md

```diff
@@ -1111,6 +1111,13 @@ added: v6.0.0
 
 Print stack traces for process warnings (including deprecations).
 
+### `--no-extra-info-on-fatal-exception`
```

Comment:
this list is sorted alphabetically. should be just after `--no-deprecation`

Reply:
Done, updated! :+1: